bin/ocr-validate.sh

```diff
@@ -79,4 +79,9 @@ while [[ "$1" = -* ]]; do
     esac
     shift
 done
+
+if [[ -d "$SHAREDIR/venv" ]];then
+    . "$SHAREDIR/venv/bin/activate"
+fi
+
```

Comment:
```suggestion

test -d "$SHAREDIR/venv" && . "$SHAREDIR/venv/bin/activate"

```

Reply:
Is the venv really required for the validation part?

Reply:
It is.